README.md

```diff
@@ -1 +1,148 @@
-# planning-evaluation-framework
\ No newline at end of file
+# Planning Evaluation Framework
+
+## Overview
+
+This repo contains the code for the Planning Evaluation Framework, which
+evaluates planning models on both synthetic and real data. A planning model
+predicts the effectiveness of an advertising campaign that is spread across
+multiple publishers, by estimating the number of people that would be reached at
+various frequencies as a function of the amount spent with each publisher. The
+purpose of the Planning Evaluation Framework is to
+
+*   Provide a means for identifying the best candidate planning models to be
+    used in the Halo measurement system;
+*   Provide the ability for publishers to evaluate the quality of models when
+    applied to their own internal data sets; and,
+*   Provide the ability to simulate the behavior of the Halo system on realistic
+    advertiser workflows.
+
+### Workflow
+
+**Note: Much of the following is aspirational and will be completed over the
+course of the project**
+
+A typical workflow consists of the following steps:
+
+1.  Generate a Data Design.
+1.  Configure an Experimental Design.
+1.  Run the Evaluator to create the results of evaluating the experiments
+    against the data.
+1.  Analyze the results in Colab.
+
+A Data Design specifies the set of data that will be used for evaluating models.
+It is given by a directory structure. Each subdirectory of the Data Design
+defines a Dataset. A Dataset in turn consists of a collection of Publisher Data
+Files. A Publisher Data File simulates the impression viewing history of a
+single publisher. It is a CSV file, where each line is of the form
+
+```
+   vid, spend
+```
+
+where vid is a virtual user id, and spend is the amount that would have to be
+spent for this impression to be shown.
+
+A Data Design can be synthetic, or it can represent actual publisher data.
+Synthetic data designs can be generated using the program
+`data_generators/synthetic_data_generator.py`. Alternatively, if you have other
+data available in your organization, you can construct your own data design. An
+example data design can be found in the directory `examples/data_designs`.
+
+An Experimental Design specifies the models that will be evaluated and the
+parameters that will be used for evaluation. An Experimental Design is given by
+a Python class. An Experimental Design is subdivided into Experiments, which are
+further broken down into Trials. An Experiment consists of a collection of
+models and model parameters that are run against a single Dataset. A Trial
+consists of simulating the outcome of one model using one Dataset. For an
```

Comment:
The outcome of one model using one Dataset.

Perhaps + under one set of experimental parameters (e.g., number of training points, total privacy budget). I feel that experimental parameters are different from model parameters.

Reply:
Let me know if you like this wording better.

Reply:
Yes I like it. Thanks!